### DISCUSSION TITLE
I'm out of options, as I cannot pass student verification! Help would be appreciated!

### DISCUSSION THREAD

**Discussion by Gregzeee** [community/community#60139] [GitHub Education] [answered]
### Hi everyone,

Hello GitHub Education Community,

I'm heavily struggling to get through the student verification process when creating an application. I currently have 11 denied applications, and tried almost everything!

I do have a school email, which I'm using for the application, except this email unfortunately isn't verified by GitHub, so it doesn't show the verified badge when selected. However, I do believe that this is still better than not having a school email address.

Besides having a school email address, I also have a School ID. The front of my school ID contains my full name, picture, Card ID, school logo, and school name, and the back of it contains a date.

The problem is that when I try to create an application on my phone I obviously can only upload one picture, so I either upload the front of my school ID or I upload the back of it. Unfortunately and completely understandable both of those options don't work. The next thing I tried is taking a picture of both sides, and editing them together, to then upload that image, but it detects that I'm no longer using my camera, and I get a message that I could not be verified and that my chances would improve once I use my camera to upload the document. I also tried to take a picture of the edited picture from my phone, but that didn't work either.

The next thing I tried was asking my sister for her school ID since she studies at the same school as I do, so I could use the front of my own ID, since that contained my personal information, and use my sister's school ID for the back, which contains a date. However this didn't work either, and my application got yet again denied.

I've read a few things about using a transcript as proof of affiliation, but I don't know how I would do that. My transcript is multiple pages long, and it's not in English. I also tried contacting GitHub Support, however, those answers seemed to be automated, as it completely skipped information I provided in a previous message within the ticket, so I gave up on that.

If anyone has an idea on how I could solve this issue I would be really grateful! 

Thank you in advance.

Kind regards,
Gregor

**Comment by GhostOf0days** [comment]
I personally verified using a screenshot of my cost of attendance bill. It should say your school name on it. If your school logo is on it, it should work.

**Reply by Gregzeee** [reply]
Hello, thank you for your quick response! However I cannot use your suggestion, since I'm younger than 18, and so my school is completely free. Therefore I don't have a cost of attendance bill. :-(

**Reply by GhostOf0days** [reply]
Please contact [Github Support](https://support.github.com/) and explain to them your situation. You can create a support ticket by clicking 'My Tickets' in the top right.

**Comment by samus-aran** [comment, marked as answer]
Hi @Gregor-Lilienberg 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. 

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
